doc/classes/String.xml

```diff
@@ -683,13 +711,23 @@
 				The splits in the returned array are sorted in the same order as the original string, from left to right.
 				If [code]maxsplit[/code] is specified, it defines the number of splits to do from the right up to [code]maxsplit[/code]. The default value of 0 means that all items are split, thus giving the same result as [method split].
 				Example:
-				[codeblock]
+				[codeblocks]
+				[gdscript]
 				var some_string = "One,Two,Three,Four"
 				var some_array = some_string.rsplit(",", true, 1)
 				print(some_array.size()) # Prints 2
 				print(some_array[0]) # Prints "Four"
 				print(some_array[1]) # Prints "Three,Two,One"
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				// There is no Rsplit...
+				//var someString = "One,Two,Three,Four";
+				//var someArray = someString.Rsplit(",", true, 1);
+				//GD.Print(someArray.Length); // Prints 2
+				//GD.Print(someArray[0]); // Prints "Four"
+				//GD.Print(someArray[1]); // Prints "Three,Two,One"
```

Comment:
Is there a built in way?

Reply:
This option seems really weird, so I checked what the existing usages in core are. It's used for `rsplit` twice in `visual_script_property_selector.cpp:474` and `thirdparty/enet/godot.cpp:315`, and for `split` thrice in `http_request.cpp:107`, `wsl_client.cpp:116`, and `wsl_server.cpp:59`. All of these usages are about detecting suffixes/prefixes (like `http:`), since `p_maxsplit` is `1`. There are zero usages where `p_maxsplit` is a value other than `0` or `1` (except in `test_string.h`).

I'm wondering if this option should be removed for 4.0 and the usages should be replaced with a more local solution. The same logic can be accomplished with `find`/`rfind` and a substring with `left`/`right`. @akien-mga @reduz